findbugs/src/test/java/edu/umd/cs/findbugs/DetectorsTest.java

```diff
@@ -237,9 +236,7 @@ private void setUpEngine(String... analyzeMe) throws IOException {
             String[] cpParts = classpath.split(File.pathSeparator);
             for (String cpStr : cpParts) {
                 File file = new File(cpStr);
-                if (file.isDirectory()) {
-                    project.addAuxClasspathEntry(file.getCanonicalPath());
-                }
+                project.addAuxClasspathEntry(file.getCanonicalPath());
```

Comment:
It works in my Eclipse.
But I have another problem: `test-harness` project introduced a cyclic dependency.

Gradle has no problem because it computes dependency based on tasks. However Eclipse computes it based on projects...

We should specify fixed version instead of `project(':test-harness')`. I'll handle this after we release 3.1.0.